pluma/pluma-app.c

```diff
@@ -653,8 +654,24 @@ is_in_viewport (PlumaWindow  *window,
 	x += vp_x;
 	y += vp_y;
 
-	sc_width = WidthOfScreen (gdk_x11_screen_get_xscreen (screen));
-	sc_height = HeightOfScreen (gdk_x11_screen_get_xscreen (screen));
+	#ifdef GDK_WINDOWING_WAYLAND
+	if (GDK_IS_WAYLAND_DISPLAY (gdk_display_get_default ()))
+	{
+		GdkMonitor *monitor = gdk_display_get_monitor(gdk_display_get_default (), 0);
+		sc_width = gdk_monitor_get_width_mm(monitor);
+		sc_height = gdk_monitor_get_height_mm(monitor);
```

Comment:
This code must work in X11 too, so, the ifdefs aren't needed.

> GdkMonitor *monitor = gdk_display_get_monitor(gdk_display_get_default (), 0);

why monitor 0? is not better `gdk_display_get_primary_monitor ()` ?

> 		sc_width = gdk_monitor_get_width_mm(monitor);
> 		sc_height = gdk_monitor_get_height_mm(monitor);

btw, `gdk_monitor_get_width_mm` and `gdk_monitor_get_height_mm` are the dimensions of the monitor in milimeters, why in milimeters? with this change it doesn't work as before

other question, what happens working with multiple monitors?